ruby/alias_manager.rb

```diff
@@ -0,0 +1,62 @@
+# ask user for their full name
+# split the string
+# swap between first and last name
+# create a loop that cheks if letter is vowel or consonant
+# 	-if vowel - create a method that finds the next vowel and call it
+# 	-else use .next to replace the consonant.
+continue = true
+new_names = []
+original_names = []
+counter = 0
+vowels = ['a', 'e', 'i', 'o', 'u']
+consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'y', 'z']
+
+def get_next(arr, letter)
+	result = if arr.include?(letter)
+		new_index = arr.index(letter)
+		if new_index == arr.length - 1
+			arr[0]
+		else
+			arr[new_index + 1]
+		end
+	end
+end
+
+while continue
+	puts "Please enter your full name:"
+	agent_name = gets.chomp
+	original_names[counter] = agent_name
+	arr_name = agent_name.split(' ')
+	agent_name = "#{arr_name[1]} #{arr_name[0]}"
+	new_name = ""
+	agent_name.each_char do |letter|
```

Comment:
Try placing this within you method.  If you want to get extra creative try making a few methods that run process.  You can call methods inside of methods as well.

`each_char` is not needed.  Because the user input is a string through `gets.chomp` they will all be chars already.  You can just use `each`.

It may not be necessary but `each_with_index` may be handy as well.